### PR TITLE
[dev] AB#35036 [Follow Ups > Sort and Filter][B.2] - Campus list is not aligned properly when you open the dropdown

### PR DESCRIPTION
**Pull request by fantaJinMode**
PR contains:
Only implementation of custom scollbar on Filter rails. But having filter rails height issues.

Need suggestion on fixing it @groberts314 @cbrewer1689  

Video Link: [https://www.loom.com/share/2a67c40ba42843989baf2007423ebc4f](https://www.loom.com/share/2a67c40ba42843989baf2007423ebc4f)

### CODE REVIEW

**Review by groberts314 — APPROVED** [review]

File: src/surfaces/filtersRail/filtersRail.jsx [hunk 1]
```diff
@@ -181,7 +186,11 @@ function FiltersRail(props) {
                         },
                     )}
                 >
-                    {children}
+                    <ScrollBar autoHide>
```
Comment (cbrewer1689) [inline src/surfaces/filtersRail/filtersRail.jsx:189]: A thought: could we setup a prop (`isScrollable`) and have logic for implementing this scrollbar?

**Review by cbrewer1689 — CHANGES_REQUESTED** [review]

File: src/surfaces/filtersRail/filtersRail.jsx (hunk 1 above)
Comment (groberts314) [reply src/surfaces/filtersRail/filtersRail.jsx:189]: Yep, I'll make it so.

Comment (groberts314) [reply src/surfaces/filtersRail/filtersRail.jsx]: Made it so in ed6c63a.

**Review by KacosPro — APPROVED** [review]

**Review by cbrewer1689 — APPROVED** [review]
:shipit:

**Comment by groberts314** [comment]
@cbrewer1689 Thanks for the merge.  What are your thoughts on a React CM UI release?  Can this go in a 10.0.2?  I think you have plans for 10.1.0 vis a vis UI Gaps, right?

**Pull request merged**